lk2nd/smp/clock-msm8953.c

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/* Copyright (c) 2022, Stephan Gerhold <stephan@gerhold.net> */
+
+#include <board.h>
+#include <boot.h>
+#include <clock.h>
+#include <platform/iomap.h>
+#include <reg.h>
+
+#define MHZ				1000000
+
+static bool platform_has_second_cluster(void)
+{
+	switch (board_platform_id()) {
+		case MSM8953:
```

Comment:
apq8053?

also are there any that don't have the second cluster? Or it's "just in case"?

Reply:
I think it just an left over from clock-msm8916.c